### ISSUE TITLE
WPS266 in similar strings template. 

### ISSUE THREAD

**Issue by levonikitatech** [wemake-services/wemake-python-styleguide#968]
# Bug report

## What's wrong

`# coding: utf-8

print('{0}{1}'.format('I ', 'am'))
print('{0}{1}'.format('We ', 'are'))
print('{0}{1}'.format('You ', 'are'))
print('{0}{1}'.format('You ', 'are'))
print('{0}{1}'.format('He, She, It ', 'is'))
print('{0}{1}'.format('They ', 'are'))
`
This code raises  "WPS226 Found string constant over-use: {0}{1}"

## How is that should be

The "clear" templates, that contains nothing but placeholders should not be interpet as constant over-use.

### flake8 information

Contents of `flake8 --bug-report`:

### pip information

Contents of `pip freeze`:
astor==0.8.0
atomicwrites==1.3.0
attrs==19.3.0
bandit==1.6.2
CacheControl==0.12.5
cached-property==1.5.1
cachetools==3.1.1
cachy==0.2.0
certifi==2019.9.11
cffi==1.13.1
chardet==3.0.4
cleo==0.6.8
colorama==0.3.9
cryptography==2.8
docutils==0.15.2
entrypoints==0.3
eradicate==1.0
flake8==3.7.9
flake8-annotations-complexity==0.0.2
flake8-bandit==2.1.2
flake8-broken-line==0.1.1
flake8-bugbear==19.8.0
flake8-builtins==1.4.1
flake8-coding==1.3.2
flake8-commas==2.0.0
flake8-comprehensions==2.3.0
flake8-debugger==3.2.1
flake8-docstrings==1.5.0
flake8-eradicate==0.2.3
flake8-executable==2.0.3
flake8-isort==2.7.0
flake8-logging-format==0.6.0
flake8-pep3101==1.2.1
flake8-polyfill==1.0.2
flake8-print==3.1.4
flake8-quotes==2.1.0
flake8-rst-docstrings==0.0.11
flake8-string-format==0.2.3
future==0.18.1
gitdb2==2.0.6
GitPython==3.0.4
google-api-python-client==1.7.11
google-auth==1.6.3
google-auth-httplib2==0.0.3
google-auth-oauthlib==0.4.1
gspread==3.1.0
html5lib==1.0.1
httplib2==0.14.0
idna==2.8
importlib-metadata==0.23
isort==4.3.21
jsonschema==3.1.1
lockfile==0.12.2
mando==0.6.4
mccabe==0.6.1
more-itertools==7.2.0
msgpack==0.6.2
oauth2client==4.1.3
oauthlib==3.1.0
pastel==0.1.1
pbr==5.4.3
pep8-naming==0.8.2
pkginfo==1.5.0.1
pluggy==0.13.0
poetry==0.12.15
py==1.8.0
pyasn1==0.4.7
pyasn1-modules==0.2.7
pycodestyle==2.5.0
pycparser==2.19
pydocstyle==4.0.1
pyflakes==2.1.1
Pygments==2.4.2
pylev==1.3.0
pyparsing==2.4.2
pyrsistent==0.14.11
pytest==3.10.1
python-dotenv==0.10.3
python-telegram-bot==12.2.0
PyYAML==5.1.2
radon==2.4.0
requests==2.22.0
requests-oauthlib==1.2.0
requests-toolbelt==0.8.0
restructuredtext-lint==1.3.0
rsa==4.0
shellingham==1.3.1
six==1.12.0
smmap2==2.0.5
snowballstemmer==2.0.0
stevedore==1.31.0
testfixtures==6.10.1
tomlkit==0.5.8
tornado==6.0.3
typing-extensions==3.7.4.1
uritemplate==3.0.0
urllib3==1.25.6
webencodings==0.5.1
wemake-python-styleguide==0.12.5
zipp==0.6.0

**Comment by sobolevn** [comment]
Why not: 

```python
template = '{0}{1}'

for first, second in your_data():
    print(template.format(first, second))
```

?

**Comment by levonikitatech** [comment]
@sobolevn  Cause that template can be in different places and functions, that are not related.

**Comment by sobolevn** [comment]
They are related in case they use the same template 🙂
